app/src/main/java/ar/com/wolox/android/example/ui/example/ExamplePresenter.kt

```diff
@@ -4,19 +4,48 @@ import ar.com.wolox.android.example.utils.UserSession
 import ar.com.wolox.wolmo.core.presenter.BasePresenter
 import javax.inject.Inject
 
+fun flattenSize(m: MutableMap<String, MutableList<String>>): Int {
```

Comment:
there is a function that does this kind of things, it is a function called `fold`. In another languages it is known as `reduce`